Training/python/2018v1/Training_v0p1.py

```diff
@@ -218,10 +205,20 @@ def compile_model(model, learning_rate):
     model.compile(loss=TauLosses.tau_crossentropy_v2, optimizer=opt, metrics=metrics, weighted_metrics=metrics)
 
 
-def run_training(train_suffix, model_name, model, data_loader, is_profile):
+def run_training(train_suffix, model_name, model, data_loader, to_profile):
+
+    gpus = tf.config.list_physical_devices('GPU')
+    if gpus:
+        try:
+            tf.config.experimental.set_virtual_device_configuration(gpus[data_loader.gpu_index],
```

Comment:
With my setup the current code still allocates memory in more than one GPU. Please, add `tf.config.set_visible_devices(gpus[data_loader.gpu_index], 'GPU')`